uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1005.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_Upgrader_Step_1005 {
```

Comment:
Instead of duplicating the data we have on the xml file and creating the option values manually, I think it would be better to simply import the xml file here. Please have a look on https://github.com/civicrm/civihr/blob/eee9c831c95cfa04fe2eec5c0321345424e5c1be/uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader.php#L33-L46 and check if we can do something similar to what is done during the installation

Reply:
Good suggestion.